### DISCUSSION TITLE
Is it possible to define multiple clusterset?

### DISCUSSION THREAD

**Discussion by adibrastegarnia** [submariner-io/submariner#2969] [Q&A] [answered]
I would like to know if it is possible to define multiple cluster sets where each cluster set consists of a few clusters rather than having one clusterset which consists of all clusters?

**Comment by tpantelis** [comment, marked as answer]
yes it is - from Submariner's viewpoint, a cluster set is basically just a broker namespace. Of course, a cluster can only be joined to one broker/cluster set. Submariner is integrated with Red Hat's [Advanced Cluster Management](https://www.redhat.com/en/technologies/management/advanced-cluster-management?sc_cid=7013a000003ScmnAAC&gad_source=1&gclid=EAIaIQobChMIv-yOgOizhQMVlpxaBR3MRAwlEAAYASAAEgJxIPD_BwE&gclsrc=aw.ds ) product which manages multiple cluster sets.

**Reply by adibrastegarnia** [reply]
@tpantelis  thanks for your response. If I understand this correctly then that means if we want to have multiple cluster sets then we need to deploy multiple brokers is this a correct understanding?

**Reply by tpantelis** [reply]
yes. In Submariner, a broker is equivalent to a cluster set.
